en/04_Changelogs/5.0.0.md

```diff
@@ -107,9 +109,11 @@ Review [Silverstripe CMS 5 fixed dependencies](/project_governance/fixed_depende
 
 - `swiftmailer/swiftmailer` has been removed and replaced with `symfony/mailer`
 - Various Symfony dependencies have been upgraded from 4.x to 6.x. A small number of code changes were made in `silverstripe/framework` to work with the symfony 6.x dependencies.
-- `silverstripe/graphql` v3 is no longer supported. It is recommended that you ensure you are using `silverstripe/graphl` v4 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. There is [documentation for the upgrade process here](https://docs.silverstripe.org/en/4/upgrading/upgrading_to_graphql_4/).
-- PHPUnit 5.7 is no longer supported. It is recommended that you ensure your tests are running with PHPUnit 9 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. There is [documentation for the upgrade process here](https://docs.silverstripe.org/en/4/upgrading/phpunit9/).
-- `masterminds/html5` is now used to parse HTML content instead of `DomDocument::loadHTML()`. This may cause slight changes in rendered content if the content being saved contains invalid HTML. The optional module `silverstripe/html5` is no longer required for rendering HTML 5 content.
+- `silverstripe/graphql` v3 is no longer supported. It is recommended that you ensure you are using `silverstripe/graphl` v4 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. 
+  - Review the [`silverstripe/graphql` v4 upgrade documentation](https://docs.silverstripe.org/en/4/upgrading/upgrading_to_graphql_4/) for more details.
+- PHPUnit 5.7 is no longer supported. It is recommended that you ensure your tests are running with PHPUnit 9 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. 
+  - Review the [PHPUnit 9 documentation ](https://docs.silverstripe.org/en/4/upgrading/phpunit9/) for more details.
+- `masterminds/html5` is now used to parse HTML content instead of `DomDocument::loadHTML()`. This may cause slight changes in rendered content if the content being saved contains invalid HTML. The `silverstripe/html5` module is no longer required for rendering HTML 5 content and should be removed.
```

Comment:
```suggestion
- `masterminds/html5` is now used to parse HTML content instead of `DomDocument::loadHTML()`. This may cause slight changes in rendered content if the content being saved contains invalid HTML. The `silverstripe/html5` module is no longer required for rendering HTML 5 content and should be removed from your project.
```

Reply:
Done